remail/email_api/service.py

```
@@ -138,8 +138,12 @@ def get_emails(self, date: datetime = None) -> list[Email]:
 
 class ImapProtocol(ProtocolTemplate):
     def __init__(
-        self, email: str, password: str, host: str, controller: "EmailController" # type: ignore
-    ):  
+        self,
+        email: str,
+        password: str,
+        host: str,
+        controller: "EmailController",  # type: ignore
```

Ruff (0.8.2)

Undefined name `EmailController`

(F821)